packages/astro/src/core/routing/manifest/create.ts

```diff
@@ -258,10 +257,12 @@ export function createRouteManifest(
 			}
 			// filter out "foo.astro_tmp" files, etc
 			if (!isDir && !validPageExtensions.has(ext) && !validEndpointExtensions.has(ext)) {
-				if (!foundInvalidFileExtensions.has(ext)) {
-					foundInvalidFileExtensions.add(ext);
-					logger.warn('astro', `Invalid file extension for Pages: ${ext}`);
-				}
+				logger.warn(
+					null,
+					`Unsupported file type ${bold(
```

Comment:
```suggestion
					`Unsupported page file type ${bold(
```
If still accurate, I do like mentioning that the file type is unsupported for use **as a page**.